src/WindowManager.vala

```diff
@@ -1290,10 +1306,10 @@ namespace Gala
 				unminimize_completed (actor);
 			if (end_animation (ref minimizing, actor))
 				minimize_completed (actor);
-			if (end_animation (ref maximizing, actor))
```

Comment:
The reason for this change is that `size_change_completed (actor);` is not needed here since this signal is always immediately emitted in the overriden `size_change` method.